Sources/Networking/Operations/PostReceiptDataOperation.swift

```diff
@@ -228,7 +228,7 @@ extension PostReceiptDataOperation.PostData: Encodable {
         try container.encodeIfPresent(self.testReceiptIdentifier, forKey: .testReceiptIdentifier)
     }
 
-    var fetchToken: String { return self.receiptData.asFetchToken }
+    var fetchToken: String { return String(data: self.receiptData, encoding: .utf8) ?? "" }
```

Comment:
It's weird that we're converting it from string to data and now back to string. Let's change the parameter in these 2 operations to `String` to avoid that.